names_mapping

```diff
@@ -0,0 +1,63 @@
+string,map to
+(including Ilex verticillata),Ilex verticillata
+(including introduced Ulmus species),Ulmus
+(including Populus deltoides),Populus deltoides
+Acer (including Acer negundo),Acer negundo
```

Comment:
I imagine in the survey, Acer negundo were identified, as well as some individuals that might be some other species of Acer

Reply:
Ok, so you'd want all Acer species to map to Acer negundo ?

Reply:
well, not exactly. I meant to discard the other unknown species. The genus is already represented, since one named species is present. I suppose a second record at the genus level could be added, but I don't think that would be very informative...